connector/connect/src/main/scala/org/apache/spark/sql/connect/dsl/package.scala

```diff
@@ -694,6 +685,18 @@ package object dsl {
           .build()
       }
 
+      def hint(name: String, parameters: Any*): Relation = {
+        Relation
+          .newBuilder()
+          .setHint(
+            Hint
```

Comment:
hmm, the proto message `Hint` was added before this PR?

Reply:
Yes, it was already added in #38857